### ISSUE TITLE
[FlightDetails.Json]: "Flight to JFK" is not defined as a heading on 'FlightDetails.Json' card.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#92]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**
1. Turn on the Talkback.
2. Navigate to the Visual.
3. Navigate to the sample card.
4. Using swipe gesture, swipe left to right in the card.
5. Navigate to FlightDetails.Json button by using swipe gesture and double tap to it.
6. FlightDetails adaptive card will be appeared.
7. Navigate to FlightDetails card by using swipe gesture and observe the issue.

**Actual Result:**
"Flight to JFK" is not defined as a heading on 'FlightDetails.Json' card. Screen reader is only announcing as, "Flight to JFK". Also, after activating the rotor setting to Heading mode by performing three fingers swipe down on perfomring Up and Down swipe focus is not landing on it.

**Expected Result:**
"Flight to JFK" label should be defined as heading so that when Talkback focus lands on it then it should announce as 'Flight to JFK heading level 2'. 

**User Impact:**
It will be confusing & difficult for the users if Heading is not defined inside the adaptive card.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/b8d9aba8-883a-401b-af75-7d91f83e1716

![Flight to JFK is not defined as a heading on 'FlightDetails Json' card](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/42e6678b-41f2-4542-959a-6b4333a84af1)

### Card JSON

```JSON
NA
```
